contracts/ETO/ETOTerms.sol

```diff
@@ -100,11 +100,15 @@ contract ETOTerms is
     uint256 public MAX_AVAILABLE_TOKENS;
     // number of tokens that can be sold in whitelist
     uint256 public MAX_AVAILABLE_TOKENS_IN_WHITELIST;
+    // equity token scale power (10^precision)
+    uint256 public EQUITY_TOKEN_POWER;
 
     // base token price in EUR-T, without any discount scheme
     uint256 private TOKEN_PRICE_EUR_ULPS;
     // equity tokens per share
     uint256 private EQUITY_TOKENS_PER_SHARE;
+    // full price fraction
```

Comment:
either the comment or the variable could be renamed for better understanding. maybe to ```DISCOUNTED_TOKEN_PRICE_EURO_ULPS``` or the comment to 
```// public-discounted price to be paid for a full token during the eto